<launch>
    <!-- 加载地图 -->
    <arg name="map" default="myenv_nav2.yaml" />
    <node name="map_server" pkg="map_server" type="map_server" args="$(find motion_planning)/map/$(arg map)"/>

    <!-- amcl -->
    <include file="$(find motion_planning)/launch/nav_amcl.launch" />

    <!-- move_base -->
    <node pkg="move_base" type="move_base" respawn="false" name="move_base" output="screen" clear_params="true">
        <rosparam file="$(find motion_planning)/param/costmap_common_params.yaml" command="load" ns="global_costmap" />
        <rosparam file="$(find motion_planning)/param/costmap_common_params.yaml" command="load" ns="local_costmap" />
        <rosparam file="$(find motion_planning)/param/local_costmap_params.yaml" command="load" />
        <rosparam file="$(find motion_planning)/param/global_costmap_params.yaml" command="load" />
        <rosparam file="$(find motion_planning)/param/base_local_planner_params.yaml" command="load" />
    </node>

    <node pkg="rviz" type="rviz" name="rviz" args="-d $(find motion_planning)/config/nav_slam.rviz"/>

    <!-- 启动运动规划节点 -->
    <!-- <node name="nav" pkg="motion_planning" type="motion_planning" /> -->
</launch>